### DISCUSSION TITLE
인터뷰 질의응답중 statistics-math-중심극한정리는 왜 유용한걸까요? 중에서

### DISCUSSION THREAD

**Discussion by stk346** [boost-devs/ai-tech-interview#145] [2. 질문] [answered]
안녕하세요~ 너무 좋은 자료 올려 주신 덕분에 잘 공부하고 있습니다! 먼저 감사하다는 말씀 드리고 싶어요 ㅎㅎ 다름이 아니라 중심극한정리 부분에서 궁금한게 있어서요.

Answer 중에서 "중심극한정리란 표본추출이 무수히 많이 수행되면(보통 30회 이상을 의미)" 라는 문장이 있는데 구글링 해보니까 약간 다른 부분이 있는 것 같아요~

https://drhongdatanote.tistory.com/57 해당 블로그에서는
"표본평균분포는 "모집단에서 표본크기가 n인 표본(예: 30개)을 여러번 반복해서 추출(예: 200번 추출)했을 때 (즉, X1(n=30), X2(n=30), X3(n=30), ... X200(n=30), 각각의 표본 평균들이 이루는 분포" 라고 적혀있어요.

추출하는 횟수는 https://intothedata.com/02.scholar_category/statistics/central_limit_theorem/ 여기서는 최소 100회라고 합니다!

따라서 "중심극한정리란 크기가 n인 표본(30개 이상)이 무수히 많이 수행되면(최소 100회 이상)" 이라고 수정하는 것이 어떨까요?

**Comment by Heeseok-Jeong** [comment, marked as answer]
좋습니다. 감사합니다!
